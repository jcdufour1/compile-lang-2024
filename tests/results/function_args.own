compile::stdout 3
hello
86
265

compile::stderr 0

compile::return_code 0

