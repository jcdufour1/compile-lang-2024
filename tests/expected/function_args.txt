hello
86
265
